<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时间分配软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>32</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>星期一</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>32</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>星期二</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>32</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>星期三</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>512</x>
      <y>32</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>星期四</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>32</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>星期五</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>752</x>
      <y>32</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>星期六</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>872</x>
      <y>32</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>星期日</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>24</y>
      <width>960</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>56</y>
      <width>960</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>672</y>
      <width>960</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>31</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>24</y>
      <width>4</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>24</y>
      <width>4</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>24</y>
      <width>4</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>24</y>
      <width>4</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>24</y>
      <width>4</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>24</y>
      <width>4</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>24</y>
      <width>4</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>24</y>
      <width>4</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>24</y>
      <width>4</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>添加事件</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>993</x>
      <y>24</y>
      <width>16</width>
      <height>648</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>680</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>生成图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>680</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存到文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Load">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>680</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>读取文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_60">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>686</y>
      <width>100</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>文件名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>680</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="type_plan">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>680</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计划</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="type_record">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>685</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Create">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>680</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>生成文件</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>680</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_61">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>686</y>
      <width>100</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>起始日期：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>起止时间</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>界面高度</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>标题</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>记录类型</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>计划保存文件</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>记录保存文件</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>计划图片文件</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>记录图片文件</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>计划生成文件</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>记录生成文件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
